ckanext/canada/tables/suppliervax.yaml

```diff
@@ -0,0 +1,424 @@
+    obligation: Optional
+    excel_required: false
+    format_type: Free text
+    validation: None
+    visible_to_public: Yes
+    occurrence: Single
+    datastore_type: text
+    excel_column_width: 35
+    form_snippet: scheming/form_snippets/textarea.html
+    form_attrs:
+      style: "width: 100%; display: block"
+
+  # 3.5
+  - datastore_id: additional_information_fr
+    label:
+      en: Additional Information (French)
+      fr: Renseignements supplémentaires (en français)
+    description:
+      en: This field will display any additional information/comments for the Briefing Note, in French.
+      fr: Ce champ affichera de plus amples renseignements/commentaires concernant la note de breffage, en français.
+    obligation: Optional
+    excel_required: false
+    format_type: Free text
+    validation: None
+    visible_to_public: Yes
+    occurrence: Single
+    datastore_type: text
+    excel_column_width: 35
+    form_snippet: scheming/form_snippets/textarea.html
+    form_attrs:
+      style: "width: 100%; display: block"
+
+
+
+  - datastore_id: record_created
+    label: Record Creation Time
+    import_template_include: false
+    visible_to_public: false
+    datastore_type: timestamp
+    preview_class: bg-info
+
+  - datastore_id: record_modified
+    label: Last Record Modification Time
+    import_template_include: false
+    visible_to_public: false
+    datastore_type: timestamp
+    preview_class: bg-info
+
+  - datastore_id: user_modified
+    label: User Last Modified Record
+    import_template_include: false
+    visible_to_public: false
+    datastore_type: text
+    preview_class: bg-info
+
+  datastore_primary_key: v_report_date
+  datastore_indexes: ""
+
+  default_preview_sort: v_report_date
+
+  excel_example_height: 32
+  excel_data_height: 32
+  excel_data_num_rows: 500
+
+  triggers:
+  - briefingt_trigger: |
+      DECLARE
+        errors text[][] := '{{}}';
+        crval RECORD;
+      BEGIN
+        errors := errors || required_error(NEW.v_report_date, 'v_report_date');
+        errors := errors || required_error(NEW.v_total, 'v_total');
+        errors := errors || required_error(NEW.v_actions, 'v_actions');
+        errors := errors || required_error(NEW.v_new_contracts, 'v_new_contracts');
+        errors := errors || required_error(NEW.v_site_inspections, 'v_site_inspections');
+        IF errors = '{{}}' THEN
+          RETURN NEW;
+        END IF;
+        RAISE EXCEPTION E'TAB-DELIMITED\t%', array_to_string(errors, E'\t');
+      END;
+  - update_record_modified_created_trigger
+
+  examples:
+    record:
+      v_report_date: 2022-02-14
+      v_total: 19
+      v_terminations: 3
+      additional_information_en: optional additional information goes here
+      additional_information_fr: text
+    filters:
+      c_report_date: 2022-02-14
+    filter_one:
+      c_report_date: 2022-02-14
```

Comment:
this filter should return a single record, how about filtering on v_report_date instead?